common-crafting.lic

```diff
@@ -142,4 +142,13 @@ module DRCC
     end
     true
   end
+  
+  def logbook_item(logbook,noun)
+    DRC.bput("get my #{logbook} logbook", "You get")
+    case bput("bundle my #{noun} with my logbook", 'You notate the', 'This work order has expired', 'The work order requires items of a higher quality')
+    when 'This work order has expired', 'The work order requires items of a higher quality'
+      dispose_trash(noun)
+    end
+    DRC.bput("stow #{logbook} logbook", 'You put')
```

Comment:
For a logbook, does it matter?  If it does then we should probably move this method back into shape.lic and sew.lic